embark/uploader/views.py

```diff
@@ -167,6 +169,14 @@ def save_file(request):
             firmware_file.file = file
             firmware_file.save()
 
+#             # not used for now since files get stored in different locations
+#             firmware_file = FirmwareFile(file=file)
+#             if(path.exists(firmware_file.get_abs_path())):
+#                 return HttpResponse("File Exists")
+#             else:
+#                 firmware_file.save()
+#                 return HttpResponse("Firmwares has been successfully saved")
```

Comment:
I commented the changes for now since we need to discuss how to handle the duplicate file upload again.
Should be enabled later on.